lib/OpenQA/Task/Job/FinalizeResults.pm

```diff
@@ -14,7 +14,7 @@
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
 package OpenQA::Task::Job::FinalizeResults;
-use Mojo::Base 'Mojolicious::Plugin';
+use Mojo::Base 'Mojolicious::Plugin', -signatures;
```

Comment:
Don't you want to convert the whole module if you're the one introducing `-signatures` into it?

Reply:
Definitely not as part of the same commit and not necessarily as part of this PR. I personally would simply convert every function at the time its parameters are changed anyways.